components/imageproc/src/processor.rs

```diff
@@ -69,6 +70,44 @@ impl ImageOp {
                 };
                 buffered_f.write_all(memory.as_bytes())?;
             }
+            Format::JXL(q) => {
+                let mut encoder = jpegxl_rs::encoder_builder();
+                if let Some(q) = q {
+                    if q == 100 {
+                        encoder.lossless(true);
+                    } else {
+                        // See `JxlEncoderDistanceFromQuality` for the formula.
```

Comment:
maybe they can expose https://libjxl.readthedocs.io/en/latest/api_encoder.html#_CPPv429JxlEncoderDistanceFromQualityf and add a method to the builder? They can't expect everyone to copy paste the formula...

Reply:
I made a PR for that: https://github.com/inflation/jpegxl-rs/pull/48

I assume it will take some time for a new release. How do you think we should handle this in the meantime?

Reply:
Let's keep the formula and we'll see when it gets released